src/geo/map/torque-layer.js

```diff
@@ -31,6 +31,10 @@ var TorqueLayer = MapLayer.extend({
   },
 
   getInfowindowData: function() {
+    var infowindow = this.get('infowindow');
```

Comment:
Something I've realised is that the layer is not using the InfowindowModel that is created [here](https://github.com/CartoDB/cartodb.js/blob/widgets/src/vis/vis.js#L1464-L1467). It probably makes more sense to move that logic to the infowindow model and use it here (instead of working with the data that comes in the viz.json). So basically we'd do something like:

```
var TorqueLayer = MapLayer.extend({

  // ...

  getInfowindowData: function() {
    this.infowindow.getFields();
  },

  // ...
})

```